xonsh/__init__.py

```diff
@@ -28,6 +26,8 @@
         _sys.modules["xonsh.platform"] = __amalgam__
         pretty = __amalgam__
         _sys.modules["xonsh.pretty"] = __amalgam__
+        xontribs_meta = __amalgam__
+        _sys.modules["xonsh.xontribs_meta"] = __amalgam__
         codecache = __amalgam__
```

Comment:
the init files change every time the amalgamate runs right? I don't think we need to commit this change

Reply:
AFAIK they don't change every time, currently I think they've changed because of the new `xonsh.completers.environment` package

Reply:
then looks alright then